tutorials/sac-intro-2-trial/sac-intro-2-trial.md

```diff
@@ -0,0 +1,71 @@
+---
+author_name: Megan Hoy
+author_profile: https://github.com/meganhoy
+title: Set up your SAP Analytics Cloud Trial Account
+description: Sign up for a free trial SAP Analytics Cloud Trial account
+auto_validation: true
+time: 5
+tags: [ tutorial>beginner, topic>cloud]
+primary_tag: software-product>sap-analytics-cloud
+---
+
+<!-- ## Prerequisites
+ - Prerequisite 1
+ - Prerequisite 2 -->
+
+## Details
+### You will learn
+  - What is included in the trial
+  - How to register an SAP.com account
+  - How to register for an SAP Analytics Cloud Trial account
+
+<!-- Add additional information: Background information, longer prerequisites -->
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](SAP Analytics Cloud Trial Overview)]
```

Comment:
Please move this to the description section, since it contains only further information, which is not part of the tutorial.

Reply:
Moved. Thanks